code/Data_Analysis.ipynb

```diff
@@ -1086,13 +1086,161 @@
     "connectome_vect=nilearn.connectome.sym_matrix_to_vec(correlation_matrices, discard_diagonal=True)\n",
     "connectome_vect"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ad1071bd",
+   "metadata": {},
+   "source": [
+    "## 4. Supervised learning algorithm"
```

Comment:
Could you maybe a few explanations (short points are ok) so that things can be evaluated respectively?